### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by Timowest12**
- Added redux dependencies.
- created slicer template.
- Created reducers for add and remove to array of objects.
- display array of objects on render and after adding.

### CODE REVIEW

**Comment by omar-labana** [comment]
Hi @Timowest12 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where the README file isn't updated to show a professional representation of the project, please update it.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
```
Comment (rloterh) [inline src/redux/configureStore.js]: - [x] Kindly note that this milestone does not require the use of `@reduxjs/toolkit`. Kindly follow the given specification. Kindly use the default `redux` package and **import** and make use of `createStore`, `combineReducers`, and _applyMiddleware_  from _redux_ in creating store.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { createSlice, current } from '@reduxjs/toolkit';
+
```
Comment (rloterh) [inline src/redux/books/books.js]: Kindly note that this milestone does not require the use of `@reduxjs/toolkit`. Kindly follow the given guidelines/ specification. Kindly ensure that `book.js` has:

- [x]  `ADD_BOOK` and `REMOVE_BOOK` as action types.
- [x] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux): `app/reducer/ACTION_TYPE`
- [x]  Sets an initial state with an empty array.
- [x] Creates and Exports **Action Creators**. (`addBook` and `removeBook`)
- [x]  Creates a reducer and exports it as default.

File: src/redux/userSlice.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
```
Comment (rloterh) [inline src/redux/userSlice.js]: - [x] You have **disabled linter check** on your code. Please enable linter check on your code to ensure code quality.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Timo, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 2 above)
Comment (Timowest12) [reply src/redux/books/books.js]: Done all of these!

File: src/redux/configureStore.js (hunk 1 above)
Comment (Timowest12) [reply src/redux/configureStore.js]: refactored code to use normal redux

File: src/redux/userSlice.js (hunk 3 above)
Comment (Timowest12) [reply src/redux/userSlice.js]: Removed all the disable file linter checks

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Timowest12,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**